src/App.jsx

```diff
@@ -264,7 +299,33 @@ function KanbanBoard() {
                                     <div 
                                       className="task-content"
                                     >
-                                      {task.content}
+                                      <div className="task-text">{task.content}</div>
+                                      {task.dueDate && (
+                                        <div 
+                                          className="task-due-date"
+                                          style={{ 
+                                            color: getDueDateColor(task.dueDate),
+                                            fontSize: '0.8em',
+                                            marginTop: '4px'
+                                          }}
```

Comment:
![kody code-review](https://img.shields.io/badge/kody-code--review-brightgreen)

```suggestion
className={`task-due-date ${getDueDateClassName(task.dueDate)}`}
```
Inline styles should be moved to a CSS file for better maintainability and reusability. Consider creating a dedicated styles file for task-related styling.



<sub>Talk to Kody by mentioning @kody</sub>

<sub>Was this suggestion helpful? React with 👍 or 👎 to help Kody learn from this interaction.</sub><!-- kody-codereview -->&#8203;
&#8203;